### PR TITLE
feat: add flag to disable version check

### LINKED ISSUE #892: New flag or env var to disable version check
Version check takes a little time as it must query github api in some circumstances its not very useful:
- integration testing of a known version
- ci when you are using docker with latest image
- you do not have external acess so the check will always fail

This would be a small optimization for these usecases.

### PR DESCRIPTION
**Pull request by elsapet**
## Description

Add flag to disable version check (for e.g. CI runs)

Closes #892

## Checklist

- [ ] I've added test coverage that shows my fix or feature works as expected.
- [ ] I've updated or added documentation if required.
- [ ] I've included usage information in the description if CLI behavior was updated or added.
- [x] PR title follows [Conventional Commits](https://www.conventionalcommits.org/) format

### CODE REVIEW

File: pkg/flag/general_flags.go [hunk 1]
```diff
@@ -29,25 +29,34 @@ var (
 		Usage:           "Load configuration from the specified path.",
 		DisableInConfig: true,
 	}
+	DisableVersionCheckFlag = Flag{
+		Name:       "disable-version-check",
+		ConfigName: "disable-version-check",
+		Value:      false,
+		Usage:      "Disable Bearer version checking",
```
Comment (cfabianski) [inline pkg/flag/general_flags.go:36]: Do you wanna use the `Hide` attribute so that it doesn’t show up?

Comment (elsapet) [reply pkg/flag/general_flags.go:36]: Happy to hide it, but I'm curious the reason why?

Comment (gotbadger) [reply pkg/flag/general_flags.go:36]: @cfabianski see #892 for use cases and why this is a useful option.

Comment (elsapet) [reply pkg/flag/general_flags.go:36]: @cfabianski are you happy to leave this one visible?

**Review by gotbadger — APPROVED** [review]

**Pull request merged**